### ISSUE TITLE
Installation Composer Installation failed

### ISSUE THREAD

**Issue by raupie** [ezsystems/LegacyBridge#72]
I downloaded the latest version of ezplatform via composer (composer create-project --no-dev ezsystems/ezplatform) and went through the install.md procedures and when I try the composer require command (composer require --update-no-dev "ezsystems/legacy-bridge:^1.0.4" "sensio/distribution-bundle:^3.0.36|^4.0.6") I'm getting a `Installation failed message`. 

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install sensio/distribution-bundle v4.0.12
    - Conclusion: don't install sensio/distribution-bundle v4.0.11
    - Conclusion: don't install sensio/distribution-bundle v4.0.10
    - Conclusion: don't install sensio/distribution-bundle v4.0.9
    - Conclusion: don't install sensio/distribution-bundle v4.0.8
    - Conclusion: don't install sensio/distribution-bundle v4.0.7
    - Conclusion: don't install sensio/distribution-bundle v4.0.6
    - Conclusion: remove sensiolabs/security-checker v4.0.0
    - Installation request for sensio/distribution-bundle ^3.0.36|^4.0.6 -> satisfiable by sensio/distribution-bundle[v3.0.36, v4.0.10, v4.0.11, v4.0.12, v4.0.6, v4.0.7, v4.0.8, v4.0.9].
    - Conclusion: don't install sensiolabs/security-checker v4.0.0
    - sensio/distribution-bundle v3.0.36 requires sensiolabs/security-checker ~3.0 -> satisfiable by sensiolabs/security-checker[v3.0.0, v3.0.1, v3.0.2].
    - Can only install one of: sensiolabs/security-checker[v3.0.0, v4.0.0].
    - Can only install one of: sensiolabs/security-checker[v3.0.1, v4.0.0].
    - Can only install one of: sensiolabs/security-checker[v3.0.2, v4.0.0].
    - Installation request for sensiolabs/security-checker (locked at v4.0.0) -> satisfiable by sensiolabs/security-checker[v4.0.0].


Installation failed, reverting ./composer.json to its original content.

```

**Comment by emodric** [comment]
Hm... This is an issue with `sensiolabs/security-checker` being locked to `4.0.0` in eZ Platform `composer.lock`, while `sensio/distribution-bundle` 3.x and 4.x only support 3.x version of Security Checker.

This can be cicumvented by manually adding `ezsystems/legacy-bridge` to `composer.json`, lowering distribution bundle requirement to `^3.0.36|^4.0.6` and running `composer update`.

Other than that, I'm not sure how to fix this properly.

The following command helps:

```
composer require --update-with-dependencies "ezsystems/legacy-bridge:^1.0.4" "sensio/distribution-bundle:^3.0.36|^4.0.6"
```

but I'm not sure if it's the right one, nor what does the warning it outputs (`Dependency "symfony/symfony" is also a root requirement, but is not explicitly whitelisted. Ignoring.`) mean.

**Comment by raupie** [comment]
Thanks @emodric, 

I modified the composer.json file and composer ran without issues. Now getting a System check issue that some folders are missing /ezpublish/{cache/logs/config} 

Added a new issue #73

**Comment by andrerom** [comment]
Solved in 7b9c8a9
